cicecore/drivers/nuopc/cmeps/ice_mesh_mod.F90

```diff
@@ -637,13 +637,10 @@ subroutine ice_mesh_check(gcomp, ice_mesh, rc)
              lon(n) = tlon(i,j,iblk)*rad_to_deg
              lat(n) = tlat(i,j,iblk)*rad_to_deg
 
-             tmplon = lon(n)
-             if(tmplon < c0)tmplon = tmplon + c360
-
              ! error check differences between internally generated lons and those read in
-             diff_lon = abs(mod(lonMesh(n) - tmplon,360.0))
+             diff_lon = abs(mod(2*c360+lonMesh(n),c360) - mod(2*c360+lon(n),c360))
```

Comment:
I have a vague memory that UFS had an issue w/ how this check was originally coded and the tmplon may be the result of that? I will make a quick test on your modification.

Reply:
Thanks!